Tools/autotest/sim_vehicle.py

```diff
@@ -877,6 +885,14 @@ def start_mavproxy(opts, stuff):
     progress("Sleeping for %f seconds" % (cmd_opts.delay_start,))
     time.sleep(float(cmd_opts.delay_start))
 
-start_mavproxy(cmd_opts, frame_infos)
+try:
+    if not cmd_opts.no_mavproxy:
+        start_mavproxy(cmd_opts, frame_infos)
+    else:
+        time.sleep(2)
```

Comment:
yes and no. I use sleep otherwise the last message isn't the following line but the message from start_vehicle function. I didn't find a better way...